FoodDeliveryUI/FoodDeliveryUI/FoodDeliveryUI.Shared/MainPage.xaml.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Java.Util;

```

Comment:
the two `java...` usings are not needed here, and prevent the build on other platforms than droid

Reply:
Done